### ISSUE TITLE
Pacific Co SO

### ISSUE THREAD

**Issue by moxboxwa** [moxboxwa/wa-pursuit-pdr-data#131]
They sent full incident report narratives

- [ ] Clarification needed from @hejokelly re: missing data files for 2019-2021
- [ ] @shiraidris007 Once uploaded, transcribe data into excel file (with dates D/M/Y only for now) and add to repo

**Comment by moxboxwa** [comment]
@hejokelly for Pacific Co the correspondence refers to files that are not on the google drive

See the correspondence file for the first request.  All installments refer to files provided -- they should all be named something like RELEASE_XXXX...., but those files are not in the folder AFAICT.  See the second request correspondence folder for examples.

**Comment by hejokelly** [comment]
> They sent full incident report narratives
> 
> * [ ]  Clarification needed from @hejokelly re: missing data files for 2019-2020
> * [ ]  @shiraidris007 Once uploaded, transcribe data into excel file (with dates D/M/Y only for now) and add to repo

Note this is actually a follow-up task for Pacific CITY, not county. The excel spreadsheet they sent only has one incident for 2019, none for 2020, and then a bunch for each following year.

**Comment by moxboxwa** [comment]
@hejokelly This is a complicated case I think b/c the correspondence shows there was an "abandoned" request.  And there's LOTS of correspondence, broken into "First" and "Second" rounds.

TLDR; we're missing 4 of the 6 incident files, and a bunch of what look like they might be policies.  If you've downloaded them, they're not in the Google drive or in the repo.  If you never downloaded them, they're probably still in the portal.

## FIRST round of correspondence:

After clarification from you, they are providing files on a total of 6 pursuit incidents,

![image](https://github.com/user-attachments/assets/dbf03935-3757-47d5-8de3-f52cede20ad7)

screenshot from [this file](https://github.com/moxboxwa/wa-pursuit-pdr-data/blob/main/County-SO-Data/Pacific/Correspondence/1st%20Request/P011970_Fulfilled-No_Cost.pdf).  Note they list a file for each year.

But in the repo, I only see the files for 2022 and 2023 (and these are also the only files on the google drive):

![image](https://github.com/user-attachments/assets/c9731515-780f-4919-848d-5ef7a850a1c6)

So, the question is whether the other files they say they made available are lying around somewhere else, or whether they were never downloaded from the portal.

The rest of the "installments" in the first correspondence folder refer to a bunch of other files they made available, almost all starting with the word "RELEASE..."  For example:

![image](https://github.com/user-attachments/assets/2f081915-ec2c-4e0f-a624-71cac5ff3f2f)

from [this file](https://github.com/moxboxwa/wa-pursuit-pdr-data/blob/main/County-SO-Data/Pacific/Correspondence/1st%20Request/P012202_1st_Installment_Letter%20(1).pdf).

I don't see those files around anywhere either.  I think they are probably policy docs.

## Second round of correspondence

All of the response files named in the correspondence are also in the folder.  But they're also uniformly useless and unresponsive.

However, they note that the files from the first request are still available for you in the document portal.

**Comment by moxboxwa** [comment]
Need a status update on this one

**Comment by moxboxwa** [comment]
@hejokelly will do a local file search for the missing files on her old computer

**Comment by hejokelly** [comment]
@moxboxwa files are now uploaded.

**Comment by moxboxwa** [comment]
@hejokelly we've got what we need now, thx.
